src/scheduler/work.rs

```diff
@@ -11,6 +11,12 @@ use std::any::{type_name, TypeId};
 /// For this case, use `WorkBucket::add_with_priority_unsync` instead.
 pub trait CoordinatorWork<VM: VMBinding>: 'static + Send + GCWork<VM> {}
 
+// Error message for borrowing `GCWorkerShared::stat`.
+#[cfg(feature = "work_packet_stats")]
+const STAT_BORROWED_MSG: &str =
```

Comment:
Two `STAT_BORROWED_MSG` constants are declared and they look identical. You can remove one if they are the same. 

I suggest the string does not say about 'concurrent GC'. There could be multiple reasons that leads to this error. You can simply state that 'we try to borrow GCWorker.stat but it was already borrowed', and explain in the comment above this string that 'we mutably borrow GCWorker.stat when doing GC work, and immutably borrow stat in harness_begin/end. If somehow, these two happen at the same time, you will see this error'.

Reply:
Will do.  They are only slightly different.